smallfile_plugin.py

```diff
@@ -128,22 +171,36 @@ def smallfile_run(params: WorkloadParams) -> typing.Tuple[str, typing.Union[Work
             "--yaml-input-file",
             smallfile_cleanup_yaml[1],
         ]
-        with open(smallfile_cleanup_yaml[1], 'w') as file:
+        with open(smallfile_cleanup_yaml[1], "w") as file:
             file.write(yaml.dump(cleanup_yaml))
         try:
-            print(subprocess.check_output(smallfile_cleanup_cmd,
-                  cwd=smallfile_dir, text=True, stderr=subprocess.STDOUT))
+            print(
+                subprocess.check_output(
+                    smallfile_cleanup_cmd,
+                    cwd=smallfile_dir,
+                    text=True,
+                    stderr=subprocess.STDOUT,
+                )
+            )
```

Comment:
I'm not sure the workflow engine will be that happy to see output via `print`. @janosdebugs ?